test/routes/testSoldierRoutes.js

```diff
@@ -0,0 +1,98 @@
+const expect = require('chai').expect;
+const XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+const testServer = require("../../start.js").server;
+const MongoClient = require('mongodb').MongoClient;
+
+const DBurl = 'mongodb://localhost:27017';
+const dbName = 'myproject';
+const solId = "tt8145643";
+const solName = "lior";
+const rank = "kama";
+const limitations = ["yeshiva"]
+const standartSol = JSON.stringify({
+  "id": solId,
+  "name": solName,
+  "rank": rank,
+  "limitations": limitations
+})
+const url = 'http://localhost:3000/soldiers';
+
+describe("Soldiers tests", function () {
+
+  before("Initialaizing db", doneBefore => {
+    MongoClient.connect(DBurl, function (dbConnectErr, client) {
+      if (dbConnectErr) return doneBefore(dbConnectErr);
+      testServer.dbClient = client;
+      testServer.collection = client.db(dbName).collection('Soldiers')
+      doneBefore();
```

Comment:
It is non-standard to call done in similar cute names
tadone was the cutest.
Kill them all 😢